schema/contractInfoSchema.json

```diff
@@ -67,13 +67,19 @@
             "type": "string",
             "enum": [
               "Blog",
-              "Twitter",
+              "CoinGecko",
+              "CoinMarketCap",
+              "Discord",
+              "Email",
```

Comment:
Thought we agreed to get rid of email?

Reply:
@connorch thoughts on this?

Reply:
Yes please remove - sorry copied this from somewhere else and forgot that was in there.